tuf-spec.md

```diff
@@ -1471,17 +1486,53 @@ it in the next step.
      order of appearance.
 
     1. If the current delegation is a multi-role delegation,
-       recursively visit each role, and check that each has signed exactly the
+       recursively visit each delegated role, and check that each has signed
+       exactly the
        same non-custom metadata (i.e., length and hashes) about the target (or
-       the lack of any such metadata).
-
-    2. If the current delegation is a terminating delegation,
+       the lack of any such metadata). Otherwise, abort the update cycle, and
+       report the failure.
+
+    2. Let DELEGATEE denote
+       the current target role DELEGATOR is delegating to.
+
+    3. **Download the DELEGATEE targets metadata file**, up to either
+       the number of bytes specified in the snapshot metadata file, or some Z
+       number of bytes. The value for Z is set by the authors of the application
+       using TUF. For example, Z may be tens of kilobytes. IF DELEGATEE cannot be
+       found, end the search and report the target cannot be found.  If
+       consistent snapshots are not used (see Section 7), then the filename used
+       to download the targets metadata file is of the fixed form FILENAME.EXT
+       (e.g., delegated_role.json).  Otherwise, the filename is of the form
+       VERSION_NUMBER.FILENAME.EXT (e.g., 42.delegated_role.json), where
+       VERSION_NUMBER is the version number of the DELEGATEE metadata file listed
+       in the snapshot metadata file.  In either case, the client MUST write the
+       file to non-volatile storage as FILENAME.EXT.
+
+    4. **Check against snapshot metadata.** The hashes (if any), and
+       version number of the new DELEGATEE metadata file MUST match the trusted
+       snapshot metadata, if any.  This is done, in part, to prevent a mix-and-match
+       attack by man-in-the-middle attackers. If the new DELEGATEE metadata file
+       does not match, abort the update cycle, and report the failure.
+
+    5. **Check for an arbitrary software attack.** The new DELEGATEE
+       metadata file MUST have been signed by a threshold of keys specified in the
+       DELEGATOR metadata file.  If the new DELEGATEE metadata file is not signed
+       as required, abort the update cycle, and report the failure.
+
+    6. **Check for a freeze attack.** The latest known time
+       should be lower than the expiration timestamp in the new DELEGATEE
+       metadata file. If so, the new DELEGATEE file becomes the trusted DELEGATEE
```

Comment:
We changed the way we describe a freeze attack check, could you make this consistent with the other checks in the spec?

"The expiration timestamp in new DELEGATEE metadata file MUST be higher than the fixed update start time."